src/Elmish.WPF/WpfProgram.fs

```diff
@@ -0,0 +1,137 @@
+﻿namespace Elmish.WPF
+
+open System.Windows
+open Microsoft.Extensions.Logging
+open Microsoft.Extensions.Logging.Abstractions
+open Elmish
+
+
+type WpfProgram<'model, 'msg> =
```

Comment:
Since we are not going to expose `Elmish.Program`, can we rename this to `Elmish.Wpf.Program`?

Reply:
I agree in principle, but unfortunately that can make the user experience worse. Consider e.g. the FileDialogs sample, where we have to `open Elmish` (in addition to `open Elmish.WPF`) because the sample uses commands. When users then type `Program.` they will see a lot of noise, because none of the `Elmish.Program` functions are relevant, only the `Elmish.WPF.Program` functions. Also it's sensitive to the order; if they `open Elmish` *after* `Open Elmish.WPF` (not that I know of any heretics who would commit such sacrilege 😜), they won't see our `mkProgram`, `withSubscription`, etc. but instead those from Elmish.

Reply:
Yes, great point.  How much to do you want to hide Elmish as an implementation detail?  Elmish.WPF could also expose its own `Cmd` type and map it to the `Cmd` type in Elmish.

Reply:
I don't see the point of doing that. I suggest letting users use the Elmish interfaces for that type.

Reply:
Ok then.  I have fine this the name `WpfProgram`.